test-integ/catalogv2/z_test.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) HashiCorp, Inc.
```

Comment:
what's the relevance of the filename `z_test.go`?

Reply:
The name is meaningless. I wanted a test-only place to put some code that is shared by the tproxy test in the next PR.

The file is also largely going to be short lived once we ship non-preview builds.

Reply:
Just put it where it next to where it's used and then move it later? It is confusing as written

Reply:
renamed